test/SlowTests/Issues/RavenDb_2239.cs

```diff
@@ -1,236 +0,0 @@
-using System;
-using System.Diagnostics;
-using System.Linq;
-using FastTests;
-using Raven.Client.Documents;
-using Raven.Client.Documents.Indexes;
-using Raven.Client.Documents.Linq;
-using Raven.Client.Documents.Transformers;
-using Xunit;
-
-namespace SlowTests.Issues
-{
-    public class RavenDb_2239 : RavenTestBase
```

Comment:
Restore candidate: this test caused request timeout issue originally, also FullLogTransformerDelay test contains an assertion for the steaming duration with transformation applied